README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+## Who am I? 
+My name is Liam Cohen. I am currently a **rising third year** in the College
+majoring in *Latin American Studies and Political Science*. I highly recommend 
+that you check out the 
+[Center for Latin American Studies](https://clas.uchicago.edu/). 
```

Comment:
If you want all the text in a single paragraph, you don't have to manually split them into separate lines. You can just let the text auto-wrap in the script editor